# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/api/FileSystemManager.cpp
# Commit: 302fdb94bf7f
# Full Hash: 302fdb94bf7f5cd817b69fc4eaaaf6ff8b62bec8
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2023-02-06 21:36:23
# Regressor Bug: 1809064
# File Overlap Count: 1
# Description:
#   Bug 1809064 - Use managed endpoints for PFileSystemAccessHandle construction; r=dom-storage-reviewers,jari
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D166342
# ==============================================================================

diff -r 100b45bcf622 -r 302fdb94bf7f dom/fs/api/FileSystemManager.cpp
--- a/dom/fs/api/FileSystemManager.cpp	Mon Feb 06 15:01:20 2023 +0000
+++ b/dom/fs/api/FileSystemManager.cpp	Mon Feb 06 15:10:34 2023 +0000
@@ -61,6 +61,11 @@
   }
 }
 
+const RefPtr<FileSystemManagerChild>& FileSystemManager::ActorStrongRef()
+    const {
+  return mBackgroundRequestHandler->FileSystemManagerChildStrongRef();
+}
+
 void FileSystemManager::RegisterPromiseRequestHolder(
     PromiseRequestHolder<BoolPromise>* aHolder) {
   mPromiseRequestHolders.AppendElement(aHolder);